js/login.js

```diff
@@ -0,0 +1,41 @@
+/* variables */
+
+let logIn = document.getElementById("login-btn");
```

Comment:
```suggestion
const logIn = document.getElementById("login-btn");
```
Instead of using let for variables that won't be reassigned, use const